Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -317,15 +332,14 @@ class DropDownMenu
     statusPopup.setAttribute('aria-atomic', 'true');
     // Do not set role=alert before it is needed prevent an alert-update during 'normal' showing of menu.
     //statusPopup.setAttribute('role', 'alert');
-    if (menuButton.attr('aria-labelledby') !== undefined)
-      statusPopup.setAttribute('aria-labelledby', menuButton.attr('aria-labelledby'));
+    if (menuButton.getAttribute ('aria-labelledby') !== undefined) // TODO: should be compared with null
```

Comment:
todo needs a ticket

Reply:
Nevermind, this was valid with jquery (`attr(...)` returns `undefined` if no value is present), `getAttribute` returns `null` however.